cogs5e/initTracker.py

```diff
@@ -700,31 +708,32 @@ def __init__(self, bot):
         await combat.final()
 
     @init.command()
-    async def effect(self, ctx, name: str, effect_name: str, *args):
+    async def effect(self, ctx, target_name: str, effect_name: str, *args):
         """Attaches a status effect to a combatant.
         [args] is a set of args that affects a combatant in combat.
+        See `!help init re` to remove effects.
         __**Valid Arguments**__
-        -dur [duration] - sets the duration of the effect, in rounds
-        conc - makes effect require conc
-        end - makes effect tick on end of turn
-        -t [target] - specifies more combatants to add this effect to
+        -dur [duration] - Sets the duration of the effect, in rounds.
+        conc - Makes the effect require concentration. Will end any other concentration effects.
+        end - Makes the effect duration tick on the end of turn, rather than the beginning.
+        -t [target] - Specifies more combatants to target, chainable (e.g., "-t or1 -t or2").
         __Attacks__
-        -b [bonus] (see !a)
-        -d [damage bonus] (see !a)
-        -attack "[hit]|[damage]|[description]" - Adds an attack to the combatant
+        -b [bonus] - Adds a bonus to hit.
+        -d [damage] - Adds additional damage.
+        -attack <"[hit]|[damage]|[description]"> - Adds an attack to the combatant. Args can be blank, no `hit` will autohit (e.g., -attack "|1d6[fire]|")
         __Resists__
-        -resist [resist] - gives the combatant resistance
-        -immune [immune] - gives the combatant immunity
-        -vuln [vulnability] - gives the combatant vulnerability
-        -neutral [neutral] - removes immune/resist/vuln
+        -resist [damage type] - Gives the combatant resistance to the given damage type.
+        -immune [damage type] - Gives the combatants immunity to the given damage type.
+        -vuln [damage type] - Gives the combatant vulnerability to the given damage type.`-custom` - Makes a custom attack with 0 to hit and base damage. Use `-b` and `-d` to add damage and to hit.
+        -neutral [damage type] - Removes the combatants immunity, resistance, or vulnerability to the given damage type.
         __General__
-        -ac [ac] - modifies ac temporarily; adds if starts with +/- or sets otherwise
-        -sb [save bonus] - Adds a bonus to saving throws"""
+        -ac [ac] - modifies ac temporarily; adds if starts with +/- or sets otherwise.
```

Comment:
as above - I think we should always use <> for tags that have arguments following